streamz/dataframe/core.py

```diff
@@ -549,8 +549,7 @@ def example(self):
         return self.root.example
 
     def reset_index(self):
-        return Window(self.root.reset_index(), n=self.n, value=self.value,
-                      with_state=self.with_state, start=self.start)
+        return Window(self.root.reset_index(), n=self.n, value=self.value)
```

Comment:
Why did this change back, was it intentional?

Reply:
Yes, I guess if one uses `reset_index` on a windowed-SDF operation, they would probably ignore the `start` state downstream anyways, so I removed it.

Reply:
I don't mind which way this goes, but I would maybe have erred on including them in the method signature 
```python
def reset_index(self, with_state=None, start=None):
    return Window(self.root.reset_index(), n=self.n, value=self.value
        with_state=with_state if with_state is not None else self.with_state, 
        start=start or self.start)
```

Reply:
I don't know how I feel about this. Having a `start` in `sdf.window(...).reset_index(start=start)` can lead to unnecessary complications downstream. One should just use the appropriate `start` state in `window()` itself, rather than in `reset_index()`. This would conform the pandas/cudf `reset_index()` API.

Reply:
OK :)
If ever it comes up that someone wants to do this, it will certainly be a corner case.